training/automl/components/text_ner/spec.yaml

```diff
@@ -1,5 +1,5 @@
 $schema: http://azureml/sdk-2-0/AutoMLComponent.json
-name: microsoft_azureml_automl_text_ner_component
+name: automl-text-ner
```

Comment:
@shouryaj can you please help to confirm naming of AutoML components. "-" will not work for registering component using CLI/SDK. But in general if we should get rid of **microsoft_azureml** prefix

Reply:
There's an ongoing discussion regarding dashes in component names in the AzureML Python SDK channel.